src/main/webapp/app/course/learning-paths/components/learning-path-student-nav/learning-path-student-nav.component.html

```diff
@@ -0,0 +1,56 @@
+<div>
+    <div class="row justify-content-between">
+        <div class="col-4">
+            <div class="row justify-content-start align-items-center">
+                @if (predecessorLearningObject()) {
+                    <div class="col-md-auto">
+                        <button id="previous-button" (click)="selectLearningObject(predecessorLearningObject()!)" type="button" class="btn btn-secondary">
+                            <span jhiTranslate="artemisApp.learningPath.navigation.previousButton"></span>
+                        </button>
+                    </div>
+                    <span class="col ps-3 text-truncate text-secondary">{{ predecessorLearningObject()!.name }}</span>
+                }
+            </div>
+        </div>
+        <div ngbDropdown class="col-4 dropdown" (openChange)="setShowNavigationOverview($event)" #navOverview="ngbDropdown">
+            @if (!isLoading()) {
+                <div type="button" class="row justify-content-center align-items-center h-100" id="navigation-overview" ngbDropdownToggle>
+                    <span class="col-md-auto fw-bold">
+                        {{ currentLearningObject()?.name }}
+                    </span>
+                    <fa-icon [icon]="faChevronDown" class="col-md-auto ps-0" />
+                </div>
+            }
+            <div ngbDropdownMenu class="mt-3 col p-0" aria-labelledby="navigation-overview">
+                @if (showNavigationOverview()) {
+                    <jhi-learning-path-student-nav-overview
+                        (onLearningObjectSelected)="navOverview.close(); selectLearningObject($event)"
+                        [learningPathId]="learningPathId()"
+                        [currentLearningObject]="currentLearningObject()"
+                    />
+                }
+            </div>
+        </div>
+        <div class="col-4">
+            <div class="row justify-content-end align-items-center">
+                @if (successorLearningObject()) {
+                    <span class="col text-end pe-3 text-truncate text-secondary">{{ successorLearningObject()!.name }}</span>
+                    <div class="col-md-auto">
+                        <button
+                            id="next-button"
+                            (click)="selectLearningObject(successorLearningObject()!)"
+                            [disabled]="!isCurrentLearningObjectCompleted()"
+                            type="button"
+                            class="btn btn-primary"
+                        >
+                            <span jhiTranslate="artemisApp.learningPath.navigation.nextButton"></span>
+                        </button>
+                    </div>
+                }
+            </div>
+        </div>
+    </div>
+    <div class="progress my-3" role="progressbar" style="height: 2px">
+        <div class="progress-bar" [ngStyle]="{ width: learningPathProgress() + '%' }"></div>
```

Comment:
You could use `[style.height.px]` and `[style.width.%]` here

Reply:
Great! much easier. Thanks